src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -512,6 +512,11 @@ export default {
 	width: 100%;
 	flex-grow: 1;
 
+	&--collapsed  .crumb:last-child {
+		min-width: 100px;
```

Comment:
Hey @raimund-schluessler , old PR, but is this really intended?
![image](https://github.com/user-attachments/assets/6330150a-714b-490d-8d1d-c8b6e0f3bdba)

Reply:
Well, I put it to prevent the last crumb to shrink to zero. But in your example, it's clearly not ideal.

Reply:
Ah, I'll see later if css magic allows that now that we are 4 years later :p